sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1825,41 +1848,72 @@ def solve_for_ild_reactions(self, value, *reactions):
             >>> E, I = symbols('E, I')
             >>> R_0, R_10 = symbols('R_0, R_10')
             >>> b = Beam(10, E, I)
-            >>> p0 = b.apply_support(0, 'roller')
+            >>> p0 = b.apply_support(0, 'pin')
```

Comment:
Is this an API change? If it used to accept 'roller' for the same effect, we should support both terms.

Reply:
This change does not have anything to do with the code or what it is possible to calculate. I changed this because this was a beam that had two roller supports. Which in theory would mean that it could just roll away. For this beam it is better to define one support as a pin and one as a roller. This does not matter for the final result, in code roller and pin support behave the same.